content/en/docs/instrumentation/go/sampling.md

```diff
@@ -1,36 +1,43 @@
 ---
 title: Sampling
 weight: 80
-cSpell:ignore: sdktrace
 ---
 
 Sampling is a process that restricts the amount of traces that are generated by
 a system. The exact sampler you should use depends on your specific needs, but
 in general you should make a decision at the start of a trace, and allow the
 sampling decision to propagate to other services.
 
-A sampler needs to be set on the tracer provider when its configured, as
-follows:
+A [`Sampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#Sampler) can
+to be set on the tracer provider when register it using the
+[`WithSampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#WithSampler)
+option, as follows:
```

Comment:
```suggestion
When you registar a [`Sampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#Sampler) 
you can set it to the tracer provider using the
[`WithSampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#WithSampler)
option as follows:
```

Reply:
I do not understand this change.

If you want to set a sampler then you have to do it via `WithSampler`

I see that my sentence was a total mess. Changing to

```suggestion
A [`Sampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#Sampler) can
be set on the tracer provider using the
[`WithSampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#WithSampler)
option, as follows:
```